corehq/apps/userreports/pillow.py

```diff
@@ -249,6 +249,7 @@ def rebuild_table(self, sql_adapter):
 def get_kafka_ucr_pillow(pillow_id='kafka-ucr-main', ucr_division=None,
                          include_ucrs=None, exclude_ucrs=None, topics=None):
     topics = topics or KAFKA_TOPICS
+    topics = [str(t) for t in topics]
```

Comment:
should switch to `kafka.util.kafka_bytestring` which is used internally by the kafka lib